src/pruner_util.cpp

```diff
@@ -208,19 +208,18 @@ bool compare_files(const IR::P4Program *prog_before,
     return before_stream->str() == after_stream->str();
 }
 
+double measure_size(const IR::P4Program *prog) {
```

Comment:
I would measure the size by writing a visitor that counts the number of statements. Much less costly.